typescript/src/e2e.test.ts

```diff
@@ -997,6 +1068,10 @@ describe("CDP Client E2E Tests", () => {
   });
 });
 
+function timeout(ms: number) {
```

Comment:
helper function to race against in those tests where the operation may hit the vitest timeout. if the vitest timeout is hit, the test will fail, even with exception handling. so we race against this function, to throw an error if test op hasnt resolved in `ms` milliseconds, and catch and log the error in that case